### DISCUSSION TITLE
How to install GitHub version?

### DISCUSSION THREAD

**Discussion by rahaaatul** [ikas-mc/ContextMenuForWindows11#158] [General]
I'm confused about the installation.

I installed the certificate but can't install the package.

![image](https://github.com/user-attachments/assets/68382985-dddf-4538-af44-ee46eb717668)

**Comment by ikas-mc** [comment]
Certificate may be imported in the wrong place

Import > Local Machine > Place all certificates in the following store > Trusted People > Finish

Install all Dependencies/*.appx then  ContextMenuCustom*

**Reply by rahaaatul** [reply]
This worked. I think you should add this to installation wiki for users who don't know how to do it (like me).

**Comment by ikas-mc** [comment]
forgot to add the installation instructions in the recent release.
I'll add a new wiki for installation
